newton/_src/usd/__init__.py

```
@@ -13,6 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+try:
+    # register the newton schema plugin
+    import newton_usd_schemas  # noqa: F401
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)